src/build.schema.json

```
@@ -37,6 +37,11 @@
           "type": "string"
         }
       },
+      "labels": {
+        "additionalProperties": {},
```

Shouldn't this be validated to be a object of strings?
```suggestion
        "additionalProperties": {
          "type": "string"
        },
```

---

Generally they should be strings, but I don't think that is required :man_shrugging: